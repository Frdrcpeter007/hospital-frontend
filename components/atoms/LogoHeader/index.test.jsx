import Page from './index';
import { render } from '@testing-library/react';

describe('LogoHeader', () => {
  it('renders correctly', () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
