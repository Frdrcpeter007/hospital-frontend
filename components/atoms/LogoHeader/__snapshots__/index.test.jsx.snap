// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoHeader renders correctly 1`] = `
<div>
  <div
    class="website-logo"
  >
    <a
      href="/"
    >
      <div
        class="logo"
        style="width: 150px; height: 50px;"
      />
    </a>
  </div>
</div>
`;
